app/controllers/groups/application_controller.rb

```diff
@@ -4,10 +4,17 @@ module Groups
   # Base Controller for Groups
   class ApplicationController < ApplicationController
     include BreadcrumbNavigation
+
+    before_action :fixed
+
     layout 'groups'
 
     private
 
+    def fixed
```

Comment:
I think we should name this to layout_fixed so when someone new cones and looks at it they have a better understanding.

Reply:
Updated in https://github.com/phac-nml/irida-next/pull/728/commits/d88b784082ed4d3c1e53a4cb9ed5e811c815f4ef